packages/commercetools/composables/src/useUser/index.ts

```diff
@@ -49,6 +50,18 @@ export default function useUser(): UseUser<Customer, any> {
     loading.value = false;
   });
 
+  const userGetters = {
```

Comment:
talk to Konrad about that

Reply:
I will help him with the helpers when his changes will be merged.

Reply:
this shouldn't be here, just something at return like
```
return {
  // props
  productGetters: makeComputedGetters(productGetters)
}
```
otherwise it's an implementation nightmare :)